apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/db/dao/ShardDaoTest.java

```diff
@@ -94,6 +99,8 @@ void setUp() {
 
     @Test
     void insertDelete() {
+        long maxId = dao.getMaxShardId();
+        assertEquals(1, maxId);
 
         Shard shard = new Shard(1L, "0000001");
```

Comment:
Hexadecimal string, which represent sha-256 hash could be better

Reply:
updated